src/plugins/data/public/ui/filter_bar/_global_filter_group.scss

```diff
@@ -55,3 +55,12 @@
     margin-top: $euiSize * -1;
   }
 }
+
+.globalFilterGroup__removeAllFilters {
+  color: $euiColorDangerText;
+}
+
+.globalFilterGroup__filterPrefix {
+  padding-top: 10px;
+  padding-right: 10px;
```

Comment:
can we use $euiSize variables? why 10 btw?

Reply:
Updated to use $euiSizeS (8px)